src/data/symbol/symbol.tsv

```diff
@@ -15,6 +15,8 @@ POS	CHAR	Reading (space separated)	description	additional description	category	m
 句読点	！？	かんたんふ . ! ！ えくすくらめーしょん びっくり	感嘆符		GENERAL
 記号	゛	だくてん " てん てんてん きごう	濁点		OTHER
 記号	゜	はんだくてん まる ' てん きごう	半濁点		OTHER
+記号	゙	だくてん " てん てんてん きごう	濁点	合成用	OTHER		U+3099
```

Comment:
As U+3099 and U+309A are modifier characters, please remove them from this PR.
Let's discuss it in a separate change.

Indeed, I'm hesitating to add them at this moment.

Reply:
I'll remove the commit including them.

Reply:
@hiroyuki-komatsu I evacuated it to a new independent local commit.
Should I create another PR for it?

Reply:
I'd prefer not to include modifier characters at this moment, although it's possible I'd change my mind in future.
For users who want to input modifier characters, I'd recommend using the user dictionary feature.

Reply:
OK I'll just back it up in my forked repo and local working directory.